JVAlertController/JVAlertController/JVPopoverPresentationController.m

```diff
@@ -26,14 +26,92 @@
 #import <objc/runtime.h>
 #import <UIKit/UIKit.h>
 
-@interface JVPopoverPresentationController : NSObject
+@interface JVPopoverPresentationController : NSObject <UIPopoverControllerDelegate>
 @property (nonatomic, strong) UIView *sourceView;
 @property (nonatomic) CGRect sourceRect;
 @property (nonatomic, strong) UIBarButtonItem *barButtonItem;
+@property (nonatomic, weak) id<UIPopoverPresentationControllerDelegate> delegate;
+@property(nonatomic, readwrite) UIEdgeInsets popoverLayoutMargins;
+@property(nonatomic, copy) UIColor *backgroundColor;
+@property(nonatomic, copy) NSArray *passthroughViews;
+@property(nonatomic, readwrite, strong) Class< UIPopoverBackgroundViewMethods > popoverBackgroundViewClass;
+@property(nonatomic, assign) UIPopoverArrowDirection permittedArrowDirections;
+@property(nonatomic, readonly) UIPopoverArrowDirection arrowDirection;
+@property(nonatomic, readwrite, weak) UIPopoverController *jv_legacyPopoverController;
 @end
 
 @implementation JVPopoverPresentationController
 
+@dynamic popoverLayoutMargins, backgroundColor, passthroughViews, popoverBackgroundViewClass, arrowDirection;
+
+- (UIEdgeInsets)getPopoverLayoutMargins {
+    return self.jv_legacyPopoverController.popoverLayoutMargins;
+}
```

Comment:
Hm, yes! This seems definitely an error. I am very sorry for that and will fix that ASAP.

I thought to have read once that the automatic created getters use "get" as prefix. Since I usually avoid dynamic getter I have done that wrong. Maybe I was too much into Java lately :)

A can't find any page stating that. Instead found "getters should have the same name as the property" here: http://www.apeth.com/iOSBook/ch12.html